src/Features/VisualBasic/Portable/AddImport/VisualBasicAddImportCodeFixProvider.vb

```diff
@@ -9,21 +9,7 @@ Imports Microsoft.CodeAnalysis.Packaging
 Imports Microsoft.CodeAnalysis.SymbolSearch
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.AddImport
-    <ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=PredefinedCodeFixProviderNames.AddImport), [Shared]>
-    Friend Class VisualBasicAddImportCodeFixProvider
-        Inherits AbstractAddImportCodeFixProvider
-
-        Public Sub New()
-        End Sub
-
-        ''' <summary>
-        ''' For testing purposes so that tests can pass in mocks for these values.
-        ''' </summary>
-        Friend Sub New(installerService As IPackageInstallerService,
-                       searchService As ISymbolSearchService)
-            MyBase.New(installerService, searchService)
-        End Sub
-
+    Friend Module AddImportDiagnosticIds
```

Comment:
any reason it got refactored this way? is AddImprotDiagnosticIds shared with other? in test ?

Reply:
ah.. it is used below. :) might consider extracting these out to its own file?

Reply:
Yeah. I refactored it to match the CSharp version because it was easier to work with. I can extract both sets of Ids to their own file.

Reply:
Note: please do code-moves in a separate PR unless absolutely necessary.  It makes reviewing PRs much easier.  Otherwise one has to manually validate nothing changed when the code moved.  THanks!

Reply:
Perhaps this code is fine where it is for now. =)